### PR TITLE
Add new Jellyfin logo to connect screen

### PR DESCRIPTION
**Pull request by nielsvanvelzen**
XML copied from ATV repository.

**Changes**

Add the new Jellyfin logo to the connect screen.

image

But in fancy Jellypurple and Jellyblue for release flavor.

### CODE REVIEW

File: app/src/main/java/org/jellyfin/mobile/ui/screens/connect/ConnectScreen.kt [hunk 1]
```diff
@@ -47,24 +41,14 @@ fun ConnectScreen(
 @Stable
 @Composable
 fun LogoHeader() {
-    CenterRow {
+    CenterRow(
+        modifier = Modifier.padding(vertical = 25.dp),
```
Comment (Maxr1998) [inline app/src/main/java/org/jellyfin/mobile/ui/screens/connect/ConnectScreen.kt:45]: Why specifically the 25 dp?

File: app/src/main/java/org/jellyfin/mobile/ui/screens/connect/ConnectScreen.kt [hunk 2]
```diff
@@ -47,24 +41,14 @@ fun ConnectScreen(
...
-            contentScale = @Suppress("MagicNumber") FixedScale(1.2f),
+                .height(72.dp),
             contentDescription = null,
         )
-        Text(
-            text = stringResource(R.string.app_name_short),
-            modifier = Modifier
-                .padding(vertical = 56.dp)
-                .padding(start = 12.dp, end = 24.dp),
-            fontFamily = FontFamily(Font(R.font.quicksand)),
```
Comment (Maxr1998) [inline app/src/main/java/org/jellyfin/mobile/ui/screens/connect/ConnectScreen.kt:65]: Is that font file still used somewhere else? If not, could we remove it?

File: app/src/main/java/org/jellyfin/mobile/ui/utils/CenterRow.kt [hunk 3]
```diff
@@ -10,9 +10,10 @@ import androidx.compose.ui.Modifier
 
 @Composable
 inline fun CenterRow(
+    modifier: Modifier = Modifier,
     content: @Composable RowScope.() -> Unit,
 ) = Row(
-    modifier = Modifier.fillMaxWidth(),
+    modifier = modifier.fillMaxWidth(),
```
Comment (Maxr1998) [inline app/src/main/java/org/jellyfin/mobile/ui/utils/CenterRow.kt:16]: I'm not 100% sure about the modifier application order, but normally, you would do size first, then padding. But it's possibly that this doesn't matter if you use `fillMaxWidth()`.

File: app/src/main/java/org/jellyfin/mobile/ui/screens/connect/ConnectScreen.kt (hunk 1 above)
Comment (nielsvanvelzen) [reply app/src/main/java/org/jellyfin/mobile/ui/screens/connect/ConnectScreen.kt:45]: Graphic design is just inputting random numbers until it feels right

File: app/src/main/java/org/jellyfin/mobile/ui/utils/CenterRow.kt (hunk 3 above)
Comment (nielsvanvelzen) [reply app/src/main/java/org/jellyfin/mobile/ui/utils/CenterRow.kt:16]: Changed to
```
 modifier = Modifier
        .fillMaxWidth()
        .then(modifier),
```

Comment (Maxr1998) [reply app/src/main/java/org/jellyfin/mobile/ui/utils/CenterRow.kt]: Perfect!

File: app/src/main/java/org/jellyfin/mobile/ui/screens/connect/ConnectScreen.kt (hunk 1 above)
Comment (Maxr1998) [reply app/src/main/java/org/jellyfin/mobile/ui/screens/connect/ConnectScreen.kt:45]: The odd number irks me, but oh well.

**Review by Maxr1998 — APPROVED** [review]

**Pull request merged**